src/main/frontend/client/assets/scss/main.scss

```diff
@@ -2,6 +2,7 @@
 @import "~foundation-sites/scss/foundation";
 @include foundation-everything;
 // place global css here
+
 *,
 *::before,
 *::after {
```

Comment:
This is also a great place to reset the default margins and padding to 0 so you can be more consistent across browsers.